src/EnergyPlus/GeneralRoutines.cc

```diff
@@ -684,12 +684,9 @@ void CheckThisAirSystemForSizing(EnergyPlusData &state, int const AirLoopNum, bo
     //       MODIFIED       na
```

Comment:
I was hesitant to remove these 3 suggestions:

[src/EnergyPlus/GeneralRoutines.cc:299]:(style),[unreadVariable],Variable 'Converged' is assigned a value that is never used.
[src/EnergyPlus/GeneralRoutines.cc:549]:(style),[unreadVariable],Variable 'Converged' is assigned a value that is never used.
[src/EnergyPlus/GeneralRoutines.cc:562]:(style),[unreadVariable],Variable 'Converged' is assigned a value that is never used.